__tests__/pet.test.js

```diff
@@ -0,0 +1,292 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+  let pet;
+  
+  beforeEach(() => {
+    pet = new Pet('Gerald');
+  });
+
+  it('returns an object', () => {
+    expect(new Pet('Gerald')).toBeInstanceOf(Object);
```

Comment:
Since you've defined pet, you can do a different assertion:
expect(pet).toBeInstanceOf(Pet);
pet - your instance; Pet - your prototype